RFC/src/RFC-0201_TariScript.md

```diff
@@ -451,90 +562,94 @@ validation steps.
 ### Standard MW transaction
 
 For this use case we have Alice who sends Bob some Tari.
-Bob's wallet is online and is able to countersign the transaction.
+Bob's wallet is  online and is able to countersign the transaction.
 
-Alice creates a new transaction spending \\( C_a \\) to a new output containing the commitment \\( C_b \\) (ignoring fees for now).
+Alice creates a new transaction spending \\( C\_a \\) to a new output containing the commitment \\( C\_b \\) (ignoring fees for now).
 
-To spend \\( C_a \\), she provides:
+To spend \\( C\_a \\), she provides:
 
-- An input that contains \\( C_a \\).
-- The script input, \\( \input_a \\).
-- A valid script signature, \\( (a*{Sa}, b*{Sa}, R*{Sa}) \\) as per (3),(4) proving that she owns the commitment
-  \\( C_a \\), knows the private key, \\( k*{Sa} \\), corresponding to \\( K\_{Sa} \\), the public key left on the stack
+* An input that contains \\( C\_a \\).
+* The script input, \\( \input_a \\).
+* A valid script signature, \\( s\_{Si} \\) as per (13),(14) proving that she owns the commitment 
+  \\( C\_a \\), knows the private key, \\( k_{Sa} \\), corresponding to \\( K_{Sa} \\), the public key left on the stack 
   after executing \\( \script_a \\) with \\( \input_a \\).
-- A sender offset public key, \\( K\_{Ob} \\).
-- The script offset, \\( \so\\) with:
-  $$
-  \begin{aligned}
-  \so  = k_{Sa} - k_{Ob}
-   \end{aligned}
-   \tag{10}
-  $$
+* A sender offset public key, \\( K_{Ob} \\).
+* The script offset, \\( \so\\) with:
+$$
+\begin{aligned}
+\so  = k_{Sa} - k_{Ob}
+\end{aligned}
+\tag{20}
+$$
 
-Alice sends the usual first round data to Bob. Bob can then completes his side of the transaction and returns his commitment
-\\(C_b\\) along with its rangeproof, the partial signature and public blinding factor back to Alice. All this happens as per the
-[standard Mimblewimble protocol].
 
-Because Alice is creating the transaction, she has a final say over the script \\( \script_b \\), similar to a
-[bitcoin transaction], but Bob can also opt to send Alice a script \\(\script_b\\) of his choosing. If Bob did not send a script,
-she chooses something akin to a `NOP` script for the script \\(\script_b\\).
+Alice sends the usual first round data to Bob. Bob can then completes his side of the transaction and returns his commitment 
+\\(C\_b\\) along with its rangeproof, the partial signature and public blinding factor back to Alice. All this happens as per the 
+[standard Mimblewimble protocol]. 
 
-Alice calculates the [sender metadata signature] \\( s\_{Mb} \\) with:
+Because Alice is creating the transaction, she has a final say over the script \\( \script_b \\), similar to a 
+[bitcoin transaction], but Bob can also opt to send Alice a script \\(\script\_b\\) of his choosing. If Bob did not send a script, 
+she chooses something akin to a `NOP` script for the script \\(\script\_b\\).
```

Comment:
How does Bob know the script that was ultimately used here?

Reply:
Good pickup; I did not fully complete this section according to the protocol described in (1) to (12).